app/scripts/controllers/account-tracker-controller.test.ts

```diff
@@ -391,14 +391,6 @@ describe('AccountTrackerController', () => {
         }).toThrow('pollingToken required');
       });
     });
-
-    it('should error if no matching pollingToken is found', async () => {
-      await withController(({ controller }) => {
-        expect(() => {
-          controller.stopPollingByPollingToken('potato');
-        }).toThrow('pollingToken not found');
-      });
-    });
   });
```

Comment:
Could you add a unit test to verify that if it is called with a token that matches nothing, and then called with a token that does match, that unsubscribe is still successful? (Basically the `'should unsubscribe from the block tracker when called with a valid polling that was the only active pollingToken for a given networkClient'`  case, but with a call to the token that doesn't match

Reply:
added in https://github.com/MetaMask/metamask-extension/pull/29707/commits/4f1f73becb738f92e47a10cbf4b0f9ea3a4906c6